Sources/MongoSwift/MongoClient.swift

```diff
@@ -304,6 +304,104 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+     /**
+      * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+      * system collections i.g. "config", "local", and "admin" databases.
```

Comment:
I think this method excludes system collections *and* those databases. system collections can be in any DB and are named like `someDB.system.something`

Reply:
same applies to other docstrings in the file